### PR TITLE
Return error from get_shipping_rates instead of silent

### PR DESCRIPTION
**Pull request by joeirimpan**
@prakashpp Please review

### CODE REVIEW

File: sale.py [hunk 1]
```diff
@@ -228,9 +228,12 @@ def get_shipping_rates(self, carriers=None, silent=False):
             carriers = Carrier.search([])
 
         rates = []
+        errors = []
```
Comment (prakashpp) [inline sale.py]: I would say messages

File: sale.py [hunk 2]
```diff
@@ -391,6 +395,7 @@ class ApplyShippingSelectRate(ModelView):
     __name__ = 'sale.sale.apply_shipping.select_rate'
 
     rate = fields.Selection([], 'Rate', required=True, sort=False)
+    message = fields.Text("Message", readonly=True)
```
Comment (prakashpp) [inline sale.py]: messages =

File: sale.py (hunk 1 above)
Comment (joeirimpan) [inline sale.py]: Fixed

File: sale.py (hunk 2 above)
Comment (joeirimpan) [inline sale.py]: fixed

**Pull request closed without merging**